tiledb/sm/compressors/dd_compressor.h

```diff
@@ -35,6 +35,8 @@
 
 #include "tiledb/common/common.h"
 
+#include <type_traits>
+
```

Comment:
Why is this needed?

Reply:
Vestigial from an intermediate state. Removed e212f5